src/bootstrap/src/utils/proc_macro_deps.rs

```diff
@@ -0,0 +1,153 @@
+/// Holds all direct and indirect dependencies of proc-macro crates in tree. See #134863
+pub static CRATES: &[&str] = &[
+    // tidy-alphabetical-start
+    "annotate-snippets",
+    "anstyle",
+    "autocfg",
```

Comment:
If we are going to add a tidy check for a list, I think it's essential to have the blessing functionality on that.